CONTRIBUTING.md

```diff
@@ -77,11 +79,14 @@ the [development mailing list](http://dev.list.galaxyproject.org/).
   request](https://help.github.com/articles/creating-a-pull-request/)
   with these changes. You pull request message ideally should include:
 
-   * A description of why the change should be made.
+   * A description of why the changes should be made.
 
-   * A description of implementation of the change.
+   * A description of the implementation of the changes.
 
-   * A description of how to test the change.
+   * A description of how to test the changes.
+
+* The pull request should pass all the continuous integration tests which are
```

Comment:
Votes are for approving/blocking a PR, this line instead is for encouraging people to look and fix their PR so that it passes the CI tests. It's like where we say that a PR "should include a description of why the changes should be made". In fact, I have added it to the "How to Contribute" section, not in "Handling Pull Requests".